### ISSUE TITLE
Username Colors does not revert on disable

### ISSUE THREAD

**Issue by MindlessPuppetz** [IFDevelopment/BeamExtended#42]
So if you disable BEx username colors via the popup settings box, it doesn't revert to proper colors. My name turns white in a stream I am not a caster/owner in.

#bugsEqualProgress!

**Comment by MindlessPuppetz** [comment]
http://puu.sh/h8Czd/67f4c948c5.jpg

**Comment by Exuviax** [comment]
Hey Mindless,

So in our first month of doing the extension we offered a deal to anyone who donated 5 or more dollars, they would receive a white username and a glow. This feature is basically default with BEx, as with the DEV Badges, it's not something that can be turned off. The was what we promised to those people, so when you donated you received that name, then you asked to have a custom username color, which makes your name that pink color. When you shut off the custom usernames, you are actually only turning off the custom username sheet - https://github.com/IFDevelopment/BeamExtended/blob/master/UsernameColors.json - and not the roles.json. 

As much as I would like to turn off the white usernames (I don't) but it is what we promised when they opted into donating. So if we were to turn it off now, it would be like if you ordered a backpack, got it, then 2 months later I cam and removed one of the straps.

Love Exu
